spec/javascripts/stimulus/side_menu_controller_test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { Application } from "stimulus";
+import side_menu_controller from "../../../app/webpacker/controllers/side_menu_controller";
+
+describe("SideMenuController", () => {
+  beforeAll(() => {
+    const application = Application.start();
+    application.register("side-menu", side_menu_controller);
+  });
+
+  describe("#side-menu", () => {
+    beforeEach(() => {
+      document.body.innerHTML = `
+        <div data-controller="side-menu" data-action="side-menu:changeActiveComponent->enterprise-form#show">
```

Comment:
It seems to be unnecessary, but I'd vote to add event name here to be more consistant. Could we add `click` :

```
click@side-menu:changeActiveComponent->enterprise-form#show
```

Does this make any sense? Not sure actually.